en/flight_modes_rover/manual_rover.md

```diff
@@ -0,0 +1,61 @@
+# Manual Mode (Rover)
+
+<img src="../../assets/site/difficulty_medium.png" title="Medium difficulty to fly" width="30px" />&nbsp;<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />&nbsp;
+
+The _Manual_ mode stops the rover when the RC control sticks are centred. To manually move/drive the vehicle you move the sticks outside of the centre.
+
+:::note
+This rover mode is enabled if you set _Manual_ mode.
+::: 
+
+(**Differential Drive Rover**) When under manual control the throttle and roll sticks control the _speed_ and _yaw rate_ of the vehicle. 
+
+(**Ackermann Drive Rover**) When under manual control the throttle and roll sticks control the _thrust_ and _torque_ of the vehicle. 
+
+## Technical Description (Differential Drive)
+
+### Manual Mode
+
+In manual mode, the left and right joystick/stick inputs directly control (open-loop) the rover's forward speed and yaw rate (rotation rate), respectively. The rover does not attempt to maintain a specific position or heading but instead responds directly to the user's inputs, without any closed-loop feedback control.
+
+![Rover Manual Sticks](../../assets/flight_modes/manual_differential_rover_new.png)
```

Comment:
Diagram looks great. What did you use to generate it? What I normally do is use online software and share the link to the editable version as an HTML comment below or sometimes I ALSO export an SVG file - since that makes editing easier.